notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<p>Is it correct to say that this is the place where the user can configure data splitting and windowing strategies (ex. parameter 'forecast_horizon')? Since these features were important enough to be mentioned in the introduction, should we list / describe the corresponding parameters in this function? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>https://cloud.google.com/vertex-ai/docs/tabular-data/data-splits#forecasting</p>

Reply:
<p>Added more descriptions around what the pipeline does and what the parameters are.</p>